KeyboardKit/KeyboardShortcuts.swift

```diff
@@ -0,0 +1,52 @@
+#if canImport(SwiftUI)
+import SwiftUI
+
+@available(iOS 14.0, *)
+public extension KeyboardShortcut {
+    
+    /// The standard keyboard shortcut for a save action, consisting of the 'S' key and the Command (⌘) modifier.
+    static let saveAction: Self = .init("s", modifiers: .command)
+    
+    /// The standard keyboard shortcut for an edit action, consisting of the 'E' key and the Command (⌘) modifier.
+    static let editAction: Self = .init("e", modifiers: .command)
+    
+    /// The standard keyboard shortcut for selecting all items, consisting of the 'A' key and the Command (⌘) modifier.
+    static let selectAllAction: Self = .init("a", modifiers: .command)
+    
+    /// The standard keyboard shortcut for a creation action, consisting of the 'N' key and the Command (⌘) modifier.
+    static let newAction: Self = .init("n", modifiers: .command)
+    
+    /// The standard keyboard shortcut for a reply action, consisting of the 'R' key and the Command (⌘) modifier.
+    static let replyAction: Self = .init("r", modifiers: .command)
+    
+    /// The standard keyboard shortcut for a refresh action, consisting of the 'R' key and the Command (⌘) modifier.
+    static let refreshAction: Self = .init("r", modifiers: .command)
+    
+    /// The standard keyboard shortcut for searching, consisting of the 'F' key and the Command (⌘) modifier.
+    static let searchAction: Self = .init("f", modifiers: .command)
+    
+    /// The standard keyboard shortcut for deletion, consisting of the Delete (⌫) key and the Command (⌘) modifier.
+    static let deleteAction: Self = .init(.delete, modifiers: .command)
+    
+    /// The standard keyboard shortcut for viewing content relating to the current day, consisting of the 'T' key and the Command (⌘) modifier.
+    static let todayAction: Self = .init("t", modifiers: .command)
+    
+    /// The standard keyboard shortcut for a zoom-in action, consisting of the plus (+) key and the Command (⌘) modifier.
+    static let zoomInAction: Self = .init("+", modifiers: .command)
+    
+    /// The standard keyboard shortcut for a zoom-out action, consisting of the minus (-) key and the Command (⌘) modifier.
+    static let zoomOutAction: Self = .init("-", modifiers: .command)
+    
+    /// The standard keyboard shortcut for an action to zoom content to its actual size, consisting of the equals (=) key and the Command (⌘) modifier.
+    static let zoomToActualSizeAction: Self = .init("=", modifiers: .command)
```

Comment:
In `KeyboardScrollView` cmd = is mapped to zoom in, because at least with English keyboard layouts I’ve seen that’s what you actually press (you don’t press cmd + because that would be shift cmd =).

The key combination I’ve been using for resetting zoom is cmd 0.

Reply:
Avoiding situations like this is one of the reasons I’m suggesting https://github.com/douglashill/KeyboardKit/pull/23/files/6ea58609730e3ece12c1d644f0968c35966d83ae#diff-543faf7a9cda9a7fbf5c1e4a1fcfdf2bc6f8105b4af02ced5922f643a2664321